venafi/resource_venafi_ssh_certificate.go

```diff
@@ -97,11 +97,21 @@ func resourceVenafiSshCertificate() *schema.Resource {
 				Computed:    true,
 			},
 			"principal": &schema.Schema{
-				Type:        schema.TypeList,
-				Description: "The requested principals.",
-				ForceNew:    true,
-				Optional:    true,
-				Elem:        &schema.Schema{Type: schema.TypeString},
+				Type:          schema.TypeList,
+				Description:   "The requested principals.",
+				ForceNew:      true,
+				Optional:      true,
+				Elem:          &schema.Schema{Type: schema.TypeString},
+				ConflictsWith: []string{"principals"},
+				Deprecated:    "This will be removed in the future. Use \"principals\" instead",
+			},
+			"principals": &schema.Schema{
```

Comment:
is there a code change to use this principals instead of principal?

Reply:
Fixed it.